.github/workflows/conformance.yml

```
@@ -136,6 +137,23 @@ jobs:
         run: |
           oasdiff breaking --fail-on ERR $BASE_SPEC $CURRENT_SPEC --match-path '^/v1/'
 
+      # Run oasdiff to detect breaking changes in the API specification when compared to the OpenAI openAPI spec
+      - name: Run OpenAPI Breaking Change Diff Against OpenAI API
+        if: steps.skip-check.outputs.skip != 'true'
+        continue-on-error: true
+        shell: bash
+        run: |
+          OPENAI_SPEC=docs/static/openai-spec.yml
+          LOCAL_SPEC=docs/static/llama-stack-spec.yaml
```

```suggestion
          LLAMA_STACK_SPEC=docs/static/llama-stack-spec.yaml
```

If you feel like this is better :)